### DISCUSSION TITLE
Error creating certificate - Command fails

### DISCUSSION THREAD

**Discussion by Yank27** [NginxProxyManager/nginx-proxy-manager#2253] [Q&A]
Hello, for several days when trying to create SSL certificates I get the following error:

Error: Command failed: certbot certonly --config "/etc/letsencrypt.ini" --cert-name "npm-39" --agree-tos --email "xxxxx@gmail.com" --domains "nc-gc.duckdns.org" --authenticator dns-duckdns --dns-duckdns-credentials "/etc/letsencrypt/credentials/credentials-39"
Saving debug log to /var/log/letsencrypt/letsencrypt.log
The resolution lifetime expired after 5.403 seconds: Server 127.0.0.11 UDP port 53 answered The DNS operation timed out.; Server 127.0.0.11 UDP port 53 answered The DNS operation timed out.; Server 127.0.0.11 UDP port 53 answered The DNS operation timed out.
Ask for help or search for solutions at https://community.letsencrypt.org. See the logfile /var/log/letsencrypt/letsencrypt.log or re-run Certbot with -v for more details.

    at ChildProcess.exithandler (node:child_process:400:12)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1093:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)

There are no ports blocked from my router and the Nginx Proxy Manager container is recreated. It is not my first SSL certificate, it would be the ninth/tenth that I create, but the first with an error.

**Comment by cma84** [comment]
Hello,

Did you manage to solve the problem?

I am in the same situation.

Regards
